app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java

```diff
@@ -27,10 +29,11 @@
 
     private CameraKitView cameraKitView;
     private static String orientationResult;
-
+    float OneEightyOverPi = 57.29577957855f;
```

Comment:
Could be better add private or final to the variable to avoid other method to modify the value

Reply:
What, I've changed it before the push, was 
`float final OneEightyOverPi = 57.29577957855f;`
Sorry I meant to set it as final... Going to push now